backend/app/services/content_machine.py

```
@@ -0,0 +1,1236 @@
+        }
+    
+    # =========================================================================
+    # ARTICLE/BLOG GENERATION
+    # =========================================================================
+    
+    async def generate_blog_post(
+        self,
+        topic: str,
+        target_length: int = 1500,
+        voice_profile_id: Optional[str] = None,
+        writer_style: Optional[str] = None,
```

Ruff (0.14.10)

Unused method argument: `writer_style`

(ARG002)